src/GRA.Domain.Service/BadgeService.cs

```diff
@@ -58,6 +58,7 @@ private string WriteBadgeFile(Badge badge, byte[] imageFile)
         public async Task<Badge> AddBadgeAsync(Badge badge, byte[] imageFile)
         {
             badge.SiteId = GetCurrentSiteId();
+            badge.AltText = badge.AltText.Trim();
```

Comment:
badge.AltText should use null coalescing before Trim() to prevent any potential null exception.

Reply:
Fixed in commit 2a6e7e46aaeb865d223b5e52d3e55e5a766d89cd